db/mysql.js

```diff
@@ -13,14 +13,20 @@ hooker.hook(require('mysql/lib/Connection').prototype, 'query', {
 	}
 });
 
-const connection = mysql.createConnection({
-	host: 'localhost',
-	user: 'root'
-});
+let connection;
+if (process.env.MYSQL_URI) {
```

Comment:
this app doesn't actually connect to mysql. if you look above we're instrumenting query to return what you pass. s you don't really need this.  were you trying to add mysql to the build or something?